apps/spruce/src/pages/version/TestAnalysis/GroupedTestMapList/FailedTestGroup/index.tsx

```diff
@@ -0,0 +1,77 @@
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import { Body } from "@leafygreen-ui/typography";
+import pluralize from "pluralize";
+import TaskStatusBadge from "@evg-ui/lib/components/Badge/TaskStatusBadge";
+import { size } from "@evg-ui/lib/constants/tokens";
+import { Accordion } from "components/Accordion";
+import { TaskBuildVariantField } from "../../types";
+import FailedTestGroupTable from "./FailedTestGroupTable";
+
+interface FailedTestGroupProps {
+  testName: string;
+  tasks: TaskBuildVariantField[];
+}
+
+const FailedTestGroup: React.FC<FailedTestGroupProps> = ({
+  tasks,
+  testName,
+}) => {
+  const statusMap = countStatuses(tasks);
+  const sortedStatuses = Object.entries(statusMap).sort((a, b) =>
+    a[0].localeCompare(b[0]),
+  );
+  return (
+    <Accordion
+      shouldRenderChildIfHidden={false}
+      title={
+        <TitleContainer>
+          <Body weight="medium">
+            {testName} failed on {tasks.length}{" "}
+            {pluralize("task", tasks.length)}
```

Comment:
is the "failed on" part superfluous given the badges have numbers? Imo it might look cleaner without this part

Reply:
I thought it would be useful if there are several different statuses in the badges. Reduces the need for some mental math. :) 
<img width="952" alt="image" src="https://github.com/user-attachments/assets/6a87eaf1-c5e2-452b-a58a-ae7e6990a757">